tests/unit/phpDocumentor/Plugin/Scrybe/Template/FactoryTest.php

```diff
@@ -40,6 +43,7 @@ public function testRegisterTemplateEngineViaConstructor()
      */
     public function testHasTwigTemplateEngine()
     {
+        $this->markTestSkipped('Invalid test case?');
```

Comment:
I see what you mean. The doc block gives the impression that there should be a default template. Would it be a good idea to adjust the Factory to hold the twig template as default?